reactnative/ReactNativeTemplateApp/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'android'
 
 dependencies {
-    compile project(':libs:SmartSync')
+    compile project(':libs:SalesforceSDK')
```

Comment:
If you were to run the template app from the SDK repo, you would be okay because the template app only does rest api calls (no smartstore or smartsync).